### DISCUSSION TITLE
Custom Spoke Video Element Mesh

### DISCUSSION THREAD

**Discussion by antpb** [mozilla/hubs#2971] [General Questions & Discussion] [answered]
Hello! I recall from the apartposters.com Hubs experience that they had a custom mesh for the 360 videos to create a sort of door into the video. Is there any way to do that within a custom Hubs Cloud instance?  Maybe it requires a custom deploy of spoke? I would greatly appreciate a point in the right direction on where I could implement that. I'd also be happy to contribute upstream and document my process for others!

**Comment by antpb** [comment, marked as answer]
I wrote a twitter thread on my learnings here: https://twitter.com/antpb/status/1303155382764711936?s=20

Will share a blog post when I can summarize it. :)

**Reply by emclaren** [reply]
Oh goodness! thanks for following up @antpb  - I was just going through our backlog of questions and was going to ask one of my colleagues to step in to see if we could get an answer for you since this didn't get responded to yet.  This looks amazing by the way!
